composer.json

```diff
@@ -11,17 +11,19 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
-    },
     "require": {
         "php": ">=5.5",
         "ext-hash": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "ext-curl": "*"
+        "ext-curl": "*",
+        "composer/ca-bundle": "~1.0",
+        "symfony/options-resolver": "~2.8|~3.0"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "~2.1",
+        "phpunit/phpunit": "^4.8 || ^5.0",
+        "monolog/monolog": "*"
     },
     "suggest": {
         "monolog/monolog": "Automatically capture Monolog events as breadcrumbs"
```

Comment:
Since now monolog is required, we should remove it from the suggestions. Should we require it as an hard dependency, BTW?